src/libs/actions/Report.ts

```diff
@@ -3429,6 +3429,10 @@ function openReportFromDeepLink(url: string) {
                                 return;
                             }
 
+                            if (isAuthenticated) {
```

Comment:
Let's add the comment that the navigation should happen only for anonymus users
Navigation for signed users is handled by linking 🙏